ci/ray_ci/macos/macos_ci.sh

```diff
@@ -11,31 +11,43 @@ export LANG="en_US.UTF-8"
 export BUILD="1"
 export DL="1"
 
+filter_flaky_tests() {
+  test_targets=""
+  while read -r line; do
+    test_targets+="$line\n"
+  done
+  # shellcheck disable=SC2086
+  echo $test_targets | bazel run ci/ray_ci/automation:filter_tests -- "flaky"
+}
```

Comment:
why are we doing bash logic again? why cannot `filter_tests` handle the input format?

Reply:
Yes that looks even cleaner! I've updated the code to pass the test targets as an arg for `filter_tests` too.